src/pages/projectSettings/tabs/GithubCommitQueueTab/getFormSchema.tsx

```diff
@@ -213,6 +220,36 @@ export const getFormSchema = (
                     enabled: {
                       enum: [true],
                     },
+                    ...(!isProduction() && {
+                      mergeQueueTitle: {
+                        title: "Merge Queue",
+                        type: "null",
+                      },
+                    }),
+                    ...(!isProduction() && {
+                      mergeQueue: {
+                        type: "string" as "string",
+                        oneOf: [
+                          {
+                            type: "string" as "string",
+                            title: "Evergreen",
+                            enum: [MergeQueue.Evergreen],
+                            description:
+                              "Use the standard commit queue owned and maintained by Evergreen.",
+                          },
+                          {
+                            type: "string" as "string",
+                            title: (
+                              <span>
+                                GitHub <BetaBadge>BETA</BetaBadge>
```

Comment:
[super nit] Probably preferable to apply `text-transform: uppercase` in the BetaBadge component.
```suggestion
                                GitHub <BetaBadge>Beta</BetaBadge>
```